qt/widgets/legacyqwt/src/MantidColorMap.cpp

```diff
@@ -23,6 +23,45 @@ namespace {
 const double LOG_ZERO_CUTOFF = 1e-15;
 } // namespace
 
+//--------------------------------------
+// Public static member functions
+//--------------------------------------
+
+/** Static convenience method for loading a color map.
+ * Points to the "colormaps.directory" mantid property
+ *
+ * @param previousFile :: last open .map file. Used to set the initial directory
+ * @param parent :: widget owner of the dialog
+ * @return QString of the filename
+ */
+QString MantidColorMap::chooseColorMap(QString previousFile, QWidget *parent) {
```

Comment:
Yes I just moved the static functions to the top of the class to keep them separate from the non-static definitions. `QString` is copy-on-write under the hold so a copy is very cheap but I agree `const &` would cause less confusion as that's what we do everywhere else.